python/google/protobuf/internal/json_format_test.py

```diff
@@ -909,6 +909,26 @@ def testParseEnumValue(self):
         'for enum type protobuf_unittest.TestAllTypes.NestedEnum.',
         json_format.Parse, '{"optionalNestedEnum": 12345}', message)
 
+  def testParseUnknownEnumStringValueProto3(self):
```

Comment:
These tests only test deserialisation. Since we are modifying code that can also *set* values, should we add tests for serialisation as well?

Reply:
I am unaware that I am changing the code that sets values.

The code changes are in the following functions:
```
_ConvertScalarFieldValue
_ConvertStructMessage
_ConvertWrapperMessage
_ConvertMapFieldValue
```

These functions are called from the following functions transitively:
_ConvertFieldValuePair
Parser::ConvertMessage
_ConvertAnyMessage
ParseDict

This is only parsing of JSON, I never change serializing to JSON or setting the values.
Did I miss something?